howto/secrets-scopes/README.md

```diff
@@ -0,0 +1,88 @@
+# Limit the secrets that can be read from secret stores 
+
+Secret stores can be configured for an app following the instructions [here](../setup-secret-store). At the same time once a secret store is configured, any secret defined within the store can be accessed by the Dapr application. 
+
+To limit the secrets that the Dapr application has access to, you can define scopes for secrets. For this the existing configuration CRD can by augmented with restrictive permissions for secret stores. 
+
+The configuration CRD is defined following the instructions [here](../../concepts/configuration/README.md).
+
+## Scenario 1 : Deny access to all secrets for a secret store
+
+Kubernetes secret store is by default added to a Dapr application, when the application is run in a Kubernetes cluster. In the scenario where it would be preferrable to deny access to the secret store for the application, either because no secrets are going to be accessed or for other security reasons, the following steps can be taken. 
```

Comment:
In Kubernetes cluster, the native Kubernetes secret store is added to Dapr application by default. In some scenarios it may be necessary to deny access to Dapr secrets for a given application. To add this configuration follow the steps below: